yarn.lock

```diff
@@ -2494,6 +2494,20 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@polkadot/metadata@npm:2.10.2-12":
```

Comment:
This is weird, it is like the resolution were not updated, so there are 2 occurences here now?

Reply:
You shouldn't need to update it at all, the current version does have the derive?